testutil/terraform_cache.go

```diff
@@ -0,0 +1,185 @@
+//go:build linux || darwin
+
+package testutil
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	// TerraformConfigFileName is the name of the Terraform CLI config file.
+	TerraformConfigFileName = "terraform.rc"
+	// CacheProvidersDirName is the subdirectory name for the provider mirror.
+	CacheProvidersDirName = "providers"
+	// CacheTemplateFilesDirName is the subdirectory name for template files.
+	CacheTemplateFilesDirName = "files"
+)
```

Comment:
If these constants aren't useful outside of this package then they should just remain private for now